### PR TITLE
Fixes #30110 - upgrade vendor to v4.7.0

### PR DESCRIPTION
**Pull request by sharvit**
Packaging PR: https://github.com/theforeman/foreman-packaging/pull/5394

### CODE REVIEW

**Review by tbrisker — COMMENTED** [review]
This works with the related pr in foreman-js. let's get that released and packaged.

**Comment by sharvit** [comment]
Thanks @tbrisker, updated to use v4.7.1

**Comment by tbrisker** [comment]
@sharvit thanks, please update packaging pr as well

File: package.json [hunk 1]
```diff
@@ -20,26 +20,21 @@
...
     "intl": "~1.2.5",
     "jed": "^1.1.1",
     "react-intl": "^2.8.0"
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@babel/preset-env": "7.9.5",
-    "@babel/plugin-transform-modules-systemjs": "7.9.0",
-    "@babel/plugin-transform-modules-commonjs": "7.9.0",
-    "@babel/plugin-transform-modules-amd": "7.9.0",
```
Comment (tbrisker) [inline package.json:33]: Following our discussion on IRC, let's keep these pinning here as well to make sure npm doesn't try to be too clever (and also the `"babel-plugin-dynamic-import-node": "2.3.0"` below).

File: package.json [hunk 2]
```diff
@@ -20,26 +20,21 @@
...
-    "@theforeman/builder": "^4.3.0",
-    "@theforeman/eslint-plugin-foreman": "^4.3.0",
-    "@theforeman/stories": "^4.3.0",
-    "@theforeman/test": "^4.3.0",
-    "@theforeman/vendor-dev": "^4.3.0",
+    "@theforeman/builder": "^4.7.1",
+    "@theforeman/eslint-plugin-foreman": "^4.7.1",
+    "@theforeman/stories": "^4.7.1",
+    "@theforeman/test": "^4.7.1",
+    "@theforeman/vendor-dev": "^4.7.1",
```
Comment (tbrisker) [inline package.json]: We've just updated packaging to 4.8.0, let's update here as well.

**Review by tbrisker — CHANGES_REQUESTED** [review]

**Comment by sharvit** [comment]
It is using 4.8.0 now

File: package.json (hunk 1 above)
Comment (tbrisker) [reply package.json:33]: Still needs addressing

**Review by tbrisker — APPROVED** [review]
Let's give it a try and see if it builds. Thanks @sharvit !

**Pull request merged**